journal/src/main/scala/com/evolutiongaming/kafka/journal/CorrelationId.scala

```diff
@@ -0,0 +1,49 @@
+package com.evolutiongaming.kafka.journal
+
+import cats.Monad
+import cats.effect.std.UUIDGen
+import cats.syntax.all.*
+import com.datastax.driver.core.{GettableByNameData, SettableData}
+import com.evolutiongaming.scassandra.*
+
+import scala.jdk.CollectionConverters.*
+
+final case class CorrelationId(value: String) extends AnyVal {
+  override def toString: String = value
+}
+
+object CorrelationId {
+  def unsafe(value: String): CorrelationId = new CorrelationId(value)
+
+  def of[F[_]: Monad: UUIDGen]: F[CorrelationId] =
+    UUIDGen[F]
+      .randomUUID
+      .map { uuid =>
+        CorrelationId(uuid.toString)
+      }
```

Comment:
just wondering if "current timestamp" or "nanoTime" would have worked too? UUID looks good, though it is much longer